04week/loop.js

```diff
@@ -0,0 +1,35 @@
+let whileCounter = 1;
```

Comment:
Need to answer the questions about how for loop is different than a while loop,
how for loop is different that for in loop, and how a while loop is different than a do while loop